### ISSUE TITLE
Include Categories in Analytics Endpoint

### ISSUE THREAD

**Issue by villanuevawill** [Bounties-Network/BountiesAPI#49]
We want to show the most used categories in our [beta analytics dashboard](https://analytics.bounties.network)

There is a bit of base work that needs to be done to first tackle this.

# Requirements
- Currently the [timeline generator](https://github.com/Bounties-Network/BountiesAPI/blob/master/bounties_api/analytics/management/commands/timeline_generator.py) aggregates the stats by schema. We need this updated to aggregate instead by platform.  Platform is a field on both bounties and fulfillments. Right now we will have `bounties-network` and `gitcoin`.  `bounties-network` should be the default if no schema is attached.
- The endpoint should show each category name along with its total count for the timeframe
- This should be completed using django calls not raw sql. It should be a query on the fly and will need to aggregate the numbers by normalized_name
- The actual endpoint however needs to join with the RankedCategory model to get the prioritized name (as the normalized name can map to multiple names, RankedCategory has already done the proper mapping work to the human readable name).

# Review
- Should correspond regularly and give an initial plan of action
- Key questions posted here
- Conversational and longer discussions should be directed to the bounties network slack in this [channel](https://join.slack.com/t/bountiesnetwork/shared_invite/enQtMzA2Mjk3MzAzODQwLTZjN2UxMmU5MWYxZTVmMmM4OGNjZDRiMDgwYTVhOTIwYmQ4MjVlMjNkZjYzOTE4MWI4OTFhOWE4ZTUzN2MyNWY)

# Qualifications
- This is a fairly advanced task, but the patterns are already in place. A good github history and good experience with python, django, SQL, docker are mandatory.

# Deadline
- Please only take this issue if you have the capacity to work on this ACTIVELY over the next week. We'd like this finished quickly and do not want someone to take it who will passively work on it.

Before starting work, please communicate your plan and availability along with your attention to take the task.  After initial communication, I'll approve the best candidate to take the task.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.2 ETH (113.25 USD @ $566.23/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Bounties-Network/BountiesAPI/49/516).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $8,537.50 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 0.2 ETH (122.18 USD @ $610.92/ETH) has been submitted by__:

 1. [@isatou](https://gitcoin.co/profile/isatou)



@villanuevawill please take a look at the submitted work:
* [PR](https://github.com/Bounties-Network/BountiesAPI/pull/54) by @isatou

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Bounties-Network/BountiesAPI/49/516)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
$21,360.59 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 0.2 ETH (120.78 USD @ $603.92/ETH) attached to this issue has been approved & issued to @isatou.__  

 * Learn more at [on the Gitcoin Issue Details page](https://gitcoin.co/issue/Bounties-Network/BountiesAPI/49/516)
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $23,871.92 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
